x-pack/plugins/security_solution/public/detection_engine/rule_monitoring/api/api_client.ts

```diff
@@ -5,6 +5,7 @@
  * 2.0.
  */
 
+import _ from 'lodash';
```

Comment:
I'm not sure if our bundler already imports only the used methods here but consider doing named imports for `omitBy` and `isUndefined`